src/app/pages/category/category.component.ts

```diff
@@ -17,19 +18,32 @@ export class CategoryComponent {
 
   private _articles!: ArticleDisplayed[];
 
+  private limit = INFINITE_SCROLL_PARAMS.limitArticles;
+  private offset = INFINITE_SCROLL_PARAMS.offsetArticles;
+
   constructor(private route: ActivatedRoute, private titleService: Title, public categoriesService: CategoriesApiService) {}
 
   ngOnInit() {
     this.route.queryParams.subscribe((params: Params) => {
       this._category = params as Category;
       this.titleService.setTitle('Sugar Rush : ' + params['name']);
-      this.categoriesService.getArticlesByCategorySlug(this._category.slug, 9, 0).subscribe((articles) => {
+      this.categoriesService.getArticlesByCategorySlug(this._category.slug, this.limit, this.offset).subscribe((articles) => {
         this._articles = articles;
-      }
-      );
+        this.limit += INFINITE_SCROLL_PARAMS.limitArticles;
+        this.offset += INFINITE_SCROLL_PARAMS.limitArticles;
+      });
     });
   };
 
+  onScroll(): void {
```

Comment:
Opional: possibility to refactor this infinite scrolls param inside the service ?